axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderLineServiceImpl.java

```diff
@@ -386,24 +386,19 @@ public SaleOrder getSaleOrder(Context context) {
 
     Context parentContext = context.getParent();
 
-    if (!parentContext.getContextClass().toString().equals(SaleOrder.class.toString())) {
-
-      parentContext = parentContext.getParent();
-    }
-
     if (parentContext == null) {
       return null;
```

Comment:
It will not work when saleOrderLine open directly in form view. Because at that time parentContext will be null.